components/Comments.jsx

```diff
@@ -0,0 +1,64 @@
+import { useEffect, useState } from "react";
+import { getCommentsByArticle, getUsers } from "./Api";
+import { Card, Button } from "react-bootstrap";
+import { FaArrowUp, FaArrowDown } from 'react-icons/fa';
+import { useParams } from "react-router-dom";
+
+const CommentList = () => {
+  const [comments, setComments] = useState([]);
+  const [users, setUsers] = useState([]);
+  let { article_id } = useParams();
+
+  useEffect(() => {
+    getCommentsByArticle(article_id).then((res) => {
+      setComments(res);
+    });
+  }, []);
+
+  return (
+    <>
+      <ul className="ul-comments">
+        {comments.map((comment) => {
+          return (
+            <li key={comment.comment_id}>
+                <div className="comment-container">
```

Comment:
Does the card need to be nested in a div as the card could very well be another div. This makes it harder for screen readers to parse for the important information